Shared/Samples/Show WFS layer with XML query/ShowWFSLayerWithXMLQueryView.swift

```diff
@@ -0,0 +1,135 @@
+// Copyright 2025 Esri
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//   https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import ArcGIS
+import SwiftUI
+
+struct ShowWFSLayerWithXMLQueryView: View {
+    /// The error shown in the error alert.
+    @State private var error: Error?
+    
+    /// Map with the Topographic basemap style.
+    @State private var map = Map(basemapStyle: .arcGISTopographic)
+    
+    /// Create a WFS (Web Feature Service) feature table using a specified URL and table name.
+    @State private var seattleTreesTable = WFSFeatureTable(
+        url: .wfsUrl, // The URL to the WFS service.
+        tableName: .seattleTreesDowntown // The name of the table within the service.
```

Comment:
```suggestion
        url: .wfsUrl,
        tableName: .seattleTreesDowntown
```
No need to add redundant comment when the variable is docstring'd.